.gitignore

```diff
@@ -209,15 +209,15 @@ tags
 /test/library/packages/HDF5/ex_lite1.h5
 /test/library/packages/HDF5/ex_lite3.h5
 /test/library/packages/ZMQ/getLastEndpointHelper
+/test/library/standard/FileSystem/chmod/dir
+/test/library/standard/FileSystem/chmod/file
```

Comment:
It looks like you missed adding the `chmod_noprivilege.good` file to the .gitignore